README.md

```diff
@@ -45,6 +45,40 @@ python3 -m http.server 8000 --directory out/Default/gen/front_end
 
 The frontend will be available at `http://localhost:8000/inspector.html` (or `http://localhost:8000/rn_inspector.html` for the RN-specific entry point).
 
+### Syncing with the upstream repo
+
+Periodically, we will sync this project with the upstream [ChromeDevTools/devtools-frontend](https://github.com/ChromeDevTools/devtools-frontend) repo. We always update our fork from a stable upstream branch.
+
+#### Viewing the last synced version
+
+```sh
+git tag --merged <branch>
+```
+
+#### Performing a repo sync
+
+1. (One-time prerequisite) Add a Git remote pointing to [ChromeDevTools/devtools-frontend](https://github.com/ChromeDevTools/devtools-frontend).
+
+    ```sh
+    git remote add chromedevtools git@github.com:ChromeDevTools/devtools-frontend.git
+    ```
+
+    > Note: The `chromedevtools` naming is optional — `upstream` will typically be set already when originally cloning your local repo with `gh repo clone`.
+
+2. Merge `main` with the target upstream branch, then resolve all conflicts(!).
```

Comment:
Realistically this should probably refer to preparing a merge branch and opening a PR, rather than working in `main`.

Reply:
Yup 👍🏻

Sidenote: I _think_ rebasing a merge commit will work well, and this therefore all works 🙏🏻.